### PR TITLE
dbc: polish the adb over DbC blog

### PR DESCRIPTION
**Pull request by ttzeng**
Signed-off-by: Pandey, Prabhat Chand <prabhat.chand.pandey@intel.com>
Signed-off-by: Tonny Tzeng <tonny.tzeng@intel.com>
Reviewed-by: Kristi Rifenbark <kristi@buzzcollectivemarketing.com>

### CODE REVIEW

**Comment by ttzeng** [comment]
@klynnrif, @prabhatchand could you help to review this entire blog? the original PR was submitted by @prabhatchand , I merged that PR and re-format with this PR, so some of paragraphs may not be presented in the diff view. Thanks.

File: source/blogs/dbc/dbc.rst [hunk 1]
```diff
@@ -42,105 +42,121 @@ is a Debug Host and the other is a Debug target. This is achieved through emulat
 a debug device by using xDBC on the debug target. The debug device presented by the
 debug target can be used by debug host for low level system debugging of target.
 
-Github PR's link for enabling adb over DbC for Celadon KBL NUC
---------------------------------------------------------------
+Github PRs for enabling adb over DbC for |C| KBL NUC
```
Comment (klynnrif) [inline source/blogs/dbc/dbc.rst]: Capitalize "hub" / GitHub 
Suggested rewrite: 
GitHub PRs to enable adb over DbC for |C| KBL NUC

File: source/blogs/dbc/dbc.rst [hunk 2]
```diff
@@ -42,105 +42,121 @@ is a Debug Host and the other is a Debug target. This is achieved through emulat
...
+#. `Adding sepolicy to access DbC RAW device node <https://github.com/projectceladon/device-androidia/pull/416>`_
 
 Steps to enable ADB over DbC support in Host Machine
 ----------------------------------------------------
 
-1. Need to edit below file, if there is no such file then create it. Make sure you
-   have root access to write to this file.
-   
-   .. image:: images/edit_udev.png
+#. Add the following permission to the udev rule */etc/udev/rules.d/51-android.rules*. Create the file if it doesn't exist.
```
Comment (klynnrif) [inline source/blogs/dbc/dbc.rst]: Suggested rewrite:
...Add the following permission to the udev rule */etc/udev/rules.d/51-android.rules* (create the file if it does not exist):

File: source/blogs/dbc/dbc.rst [hunk 3]
```diff
@@ -42,105 +42,121 @@ is a Debug Host and the other is a Debug target. This is achieved through emulat
...
 
+    .. code-block:: text
 
-2. Add below line in above file.
-   
-   .. image:: images/udev_rule.png
+        #DBC
+        SUBSYSTEM=="usb", ATTRS{idVendor}=="1d6b", ATTRS{idProduct}=="0010", MODE="0666", GROUP="plugdev", SYMLINK+="android%n"
 
+#. Reboot the host system, or run below commands with root permission to take effect.
```
Comment (klynnrif) [inline source/blogs/dbc/dbc.rst]: Suggested rewrite: 
Reboot the host system, or run the following commands with root permission to take effect:

File: source/blogs/dbc/dbc.rst [hunk 4]
```diff
@@ -42,105 +42,121 @@ is a Debug Host and the other is a Debug target. This is achieved through emulat
...
 
-3. Either reboot the Host System, or run below command in super user to allow the
-   changes to take effec.
+    .. code-block:: bash
 
-   .. image:: images/trigger_udev.png
+        root@intel:~# udevadm control --reload-rules
+        root@intel:~# udevadm trigger
 
+#. Rename or remove the kernel module *usb_debug.ko* in the host system if any.
```
Comment (klynnrif) [inline source/blogs/dbc/dbc.rst]: Replace period with colon: 
Rename or remove the kernel module *usb_debug.ko* in the host system if any:

File: source/blogs/dbc/dbc.rst [hunk 5]
```diff
@@ -42,105 +42,121 @@ is a Debug Host and the other is a Debug target. This is achieved through emulat
...
 
-4. Rename or remove the usb_debug.ko driver in the Host System if available.
+    .. code-block:: bash
 
-   .. image:: images/rename_driver.png
+        root@intel:~# cd /lib/modules/$(uname -r)/kernel/drivers/usb/serial/ && mv usb_debug.ko usbdebug
 
+#. Check whether the *usb_debug* driver module is loaded to the kernel with the following command:
```
Comment (klynnrif) [inline source/blogs/dbc/dbc.rst]: Suggested rewrite: 
Check if the *usb_debug* driver module is loaded to the kernel with the following command:

File: source/blogs/dbc/dbc.rst [hunk 6]
```diff
@@ -42,105 +42,121 @@ is a Debug Host and the other is a Debug target. This is achieved through emulat
...
 
-5. Unload usb_debug driver module if it's already loaded.
+    .. code-block:: bash
 
-   .. image:: images/unload_driver.png
+        root@intel:~# lsmod | grep usb_debug
 
+#. Unload the *usb_debug* driver module if it's already loaded.
```
Comment (klynnrif) [inline source/blogs/dbc/dbc.rst]: Suggested rewrite: 
Unload the *usb_debug* driver module if it is loaded:

File: source/blogs/dbc/dbc.rst [hunk 7]
```diff
@@ -42,105 +42,121 @@ is a Debug Host and the other is a Debug target. This is achieved through emulat
...
+        root@intel:~# setprop persist.vendor.sys.usb.adbover dbc
 
-3. Now restart the target system.
+Connect the Target to the Host System
+-------------------------------------
 
-Connection between Host and Target Machine
-------------------------------------------
+Plug the debug Target to the Host system using a `USB Type-A to Type-A (3.0) SuperSpeed Debug cable <https://www.datapro.net/products/usb-3-0-super-speed-a-a-debugging-cable.html>`_. 
+USB 2.0 Type-A to Type-A cable doesn't work in this case.
```
Comment (klynnrif) [inline source/blogs/dbc/dbc.rst]: Suggested rewrite: 
A USB 2.0 Type-A to Type-A cable does not work in this case.

File: source/blogs/dbc/dbc.rst [hunk 8]
```diff
@@ -42,105 +42,121 @@ is a Debug Host and the other is a Debug target. This is achieved through emulat
...
+        S:  SerialNumber=DW1724778700007
+        C:* #Ifs= 1 Cfg#= 1 Atr=c0 MxPwr=  0mA
+        I:* If#= 0 Alt= 0 #EPs= 2 Cls=dc(unk. ) Sub=02 Prot=01 Driver=usbfs
+        E:  Ad=01(I) Atr=02(Bulk) MxPS=1024 Ivl=0ms
+        E:  Ad=81(I) Atr=02(Bulk) MxPS=1024 Ivl=0ms
 
-   .. image:: images/sys_usb_devices.png
-   
-.. Note:: Speed should be 5000 (i.e. Spd=5000) and Driver should be usbfs (i.e. Driver=usbfs).
+.. Note:: **Speed** should be *5000* (i.e. Spd=5000) and **Driver** should be *usbfs* (i.e. Driver=usbfs) in the above command output.
```
Comment (klynnrif) [inline source/blogs/dbc/dbc.rst]: above / previous: 
**Speed** should be *5000* (i.e. Spd=5000) and **Driver** should be *usbfs* (i.e. Driver=usbfs) in the previous command output.

File: source/blogs/dbc/dbc.rst [hunk 9]
```diff
@@ -1,34 +1,34 @@
...
+.. contents:: Table of Contents
     :depth: 2
     :local:
 
 Introduction
 ------------
 
 Android Host ecosystem consists of new applications like Android Automotive,
-or Chromebook or gateway systems running on NUC like box.  One of the main
+Chromebook, or gateway running on |NUC| like systems.  One of the main
```
[the remark this answers is not in the document]
Comment (ttzeng) [reply source/blogs/dbc/dbc.rst]: Your rewriting always look good! :)

**Pull request merged**